client/components/jetpack/connection-health/index.tsx

```diff
@@ -45,6 +56,75 @@ export const JetpackConnectionHealthBanner = ( { siteId }: Props ) => {
 		);
 	}
 
+	if ( errorType === FATAL_ERROR ) {
+		return (
+			<ErrorNotice
+				errorType={ errorType }
+				errorText={ translate(
+					"Jetpack is unable to communicate with your site due to a critical error that has occurred there. Please check your site admin's email inbox for instructions."
```

Comment:
```suggestion
					"Jetpack is unable to communicate with your site due to a critical error on the site."
```

I think we should drop the "Please check your site admin's email inbox" for a few reasons:
1. We're already directing them to 'Learn how to fix'
2. They may or may not have access to the site admin's inbox.
3. Depending on the fatal, the email may or may not actually be sent. For example, with the fatal in `wp-config.php`, the email isn't sent because the fatal error trapper isn't registered yet.

Also worth noting: for WordPress.com on Atomic sites, we can point them to the 'PHP logs' tab on the new 'Site Monitoring' page for visibility into what error is.

Reply:
Instead of saying "Jetpack is unable to communicate with your site due to a critical error on the site." I'd simply say: "Jetpack can't communicate with your site due to a critical error."

Reply:
As we agreed on Slack, let's leave the current one to say where the fatal error occurred clearly.